Automattic/wp-calypso
# Seller Experience: Remove translations file

#### Changes proposed in this Pull Request

This PR removes the translations file used while developing the Seller Experience flow.

#### Testing instructions
1. Apply this patch.
2. Go through the Seller flow and verify it works correctly.

Closes https://github.com/Automattic/wp-calypso/issues/60557
